pom.xml

```diff
@@ -54,6 +54,8 @@
     <artifactsToDownload>${project.groupId}:SonarAnalyzer.CSharp:nupkg,${project.groupId}:SonarAnalyzer.VisualBasic:nupkg</artifactsToDownload>
     <!-- We are ignoring java doc warnings - this is because we are using JDK 11. Ideally we should not do that. -->
     <doclint>none</doclint>
+    <sonar.analyzer.commons>1.14.1.690</sonar.analyzer.commons>
+    <sonar.version>7.9</sonar.version>
```

Comment:
note: I tried with 8.x (latest and even 8.0) and they removed the `import org.sonar.api.internal.google.common.annotations.VisibleForTesting;` API

I will ask if it's a problem at this point or we can ship it.

Reply:
We have 
```
    <sonar.version>7.9</sonar.version>
    <sonarLTS.version>7.9</sonarLTS.version>
```
I think we should update them both after LTS release

Reply:
Why not to move `sonarLTS.version` as well when moving `sonar.version`? They were next to each other

Reply:
sonarLTS isn't in sonar-dotnet-shared-library, so I didn't move it in the parent.